tests/parm/nems.configure.cpld_esmfthreads.IN

```diff
@@ -0,0 +1,148 @@
+#############################################
+####  NEMS Run-Time Configuration File  #####
+#############################################
+
+# ESMF #
+logKindFlag:            ESMF_LOGKIND_MULTI
+globalResourceControl:  true
+
+# EARTH #
+EARTH_component_list: MED ATM CHM OCN ICE WAV
+EARTH_attributes::
+  Verbosity = 0
+::
+
+# MED #
+MED_model:                      @[med_model]
+MED_petlist_bounds:             @[med_petlist_bounds]
+MED_omp_num_threads:            @[med_omp_num_threads]
+
+# ATM #
+ATM_model:                      @[atm_model]
+ATM_petlist_bounds:             @[atm_petlist_bounds]
+ATM_omp_num_threads:            @[atm_omp_num_threads]
+ATM_attributes::
+  Verbosity = 0
+  DumpFields = false
+  ProfileMemory = false
+  OverwriteSlice = true
```

Comment:
Could we finally clean up (remove) the Verbosity and OverwriteSlice Attributes from the component attributes in the nems.configure files?  I think we could also remove DumpFields but I'll need to check how/if that is still used in any of the caps.

Reply:
Sure.  I see "DumpFields" are still used in fv3_cap.F90. Since we don't update fv3 in this PR, maybe we can remove it in other PR involving FV3 changes.